http3/server.go

```diff
@@ -618,11 +695,54 @@ func (s *Server) Close() error {
 	return err
 }
 
+func (s *Server) connectionsCount() int {
```

Comment:
The function name suggests that this function doesn't have any side effects, but it clearly does.